airflow/models/dag.py

```diff
@@ -1702,12 +1708,26 @@ def deactivate_deleted_dags(cls, alive_dag_filelocs: List[str], session=None):
             raise
 
 
-# Stringified DAGs and operators contain exactly these fields.
+class DagContext:
```

Comment:
I would like to have some more description of this. :)

Reply:
Done :)/